scene/gui/texture_progress_bar.cpp

```diff
@@ -608,11 +608,10 @@ int TextureProgressBar::get_fill_mode() {
 }
 
 void TextureProgressBar::set_radial_initial_angle(float p_angle) {
-	while (p_angle > 360) {
-		p_angle -= 360;
-	}
-	while (p_angle < 0) {
-		p_angle += 360;
+	ERR_FAIL_COND_MSG(!Math::is_finite(p_angle), "Angle is non-finite.");
+
+	if (p_angle < 0.0 || p_angle > 360.0) {
```

Comment:
```suggestion
	if (p_angle < 0.0 || p_angle >= 360.0) {
```

Reply:
This one will break editor slider when trying to set 360 degrees if I understand correctly.

Reply:
Indeed.
Technically 360 should wrap to 0, so maybe the property range should be adjusted to be below 360. That would be odd though.
Maybe it's fine like this.

Reply:
I added testcase to ensure that range is `[0.0f; 360.0f]`